### PR TITLE
Previous team member lists are End date descending order.

### PR DESCRIPTION
**Pull request by RohitNalePUCSD**
The previous team member list should be in the order of End date descending.

### CODE REVIEW

**Comment by RohitNalePUCSD** [comment]
[List order of previous team members](https://trello.com/c/z6TsF0hy/134-list-order-of-previous-team-members)

File: app/models/user_project.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ class UserProject
   scope :active_users, ->{where(:user_id.in => User.approved.pluck(:id), :active => true)}
   scope :inactive_users, ->{where(:user_id.in => User.approved.pluck(:id), :active => false)}
   scope :ex_users, ->{where(:user_id.nin => User.approved.pluck(:id))}
+  scope :inactive_and_ex_users, ->{where("$or" => [{:user_id.in => User.approved.pluck(:id), :active => false},{:user_id.nin => User.approved.pluck(:id)}])}
```
Comment (Sameer-Tilak) [inline app/models/user_project.rb:32]: We need ex-users of the project irrespective of whether they are active or resigned. Should we need above scope to get that data. I think only active = false and the project id is sufficient to get this list.

Comment (RohitNalePUCSD) [reply app/models/user_project.rb]: Done!!

**Comment by Sameer-Tilak** [comment]
LGTM, please deploy

**Pull request closed without merging**